examples/expr4.py

```diff
@@ -0,0 +1,5 @@
+def main0():
+    a: list[i32]
+    a.append(52)
```

Comment:
I think CPython won't support this. This will require an initialization of list first:
```py
a: list[i32] = []
a.append(52)
```